telemetry/metrics.yaml

```diff
@@ -1,24 +1,270 @@
----
-# Schema
 $schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
 
-page:
-  load:
+moso_events: # If we're using a single metrics.yaml across Mozilla Social, what do I call the category? Can I omit a category?
+
+  # First we have some event properties that are re-used across all events
+  # I implemented these as uuid/string/quantity metrics, instead of nesting them as extra_keys inside events. Is this valid?
+
+  # Is it possible to attach them to the baseline ping, as I've done?
+
+  mastodon_account_id:
+    type: string
+    lifetime: application
+    description: >
+      The user's account ID from Mastodon.
+    bugs:
+      - TBD
+    data_reviews:
+      - TBD
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - kdemtchouk@mozilla.com
+    expires: never
+    send_in_pings:
+      - events
+      - glean_internal_info
+
+  mastodon_handle:
```

Comment:
Should the `mastodon_account_id` by itself be enough here? Or is there additional information we get from the handle that the account_id does not provide?

Reply:
For web it probably doesnt matter since its always mozilla.social.

For mobile, if a user does not use our instance, will you want/need their handle?

Reply:
Hmm, good question. I guess this gets into whether we have (or can get) the data to _infer_ a user's handle from their 3rd party instance account ID.

I am tempted to include it (because I am not confident how we'll do that inference exercise I alluded to).